### PR TITLE
update recipe and version, set to new upstream

### PR DESCRIPTION
**Pull request by sarnold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Cleans up: pep517 packaging, python version support, tests/coverage

Note I am also the "upstream" guy for this package.

### CODE REVIEW

**Comment by sarnold** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,11 +41,14 @@ test:
         - pystache-test
 
 about:
-    home: http://github.com/defunkt/pystache
+    home: http://github.com/sarnold/pystache
```
Comment (ocefpaf) [inline recipe/meta.yaml:44]: @sarnold building from forks is usually frowned upon. What is the story here? Is the project dead? Could your changes be a patch instead? (Patches are part of the recipe and easier to inspect.)

Comment (sarnold) [reply recipe/meta.yaml:44]: Sadly, no.

* upstream no activity on github at all
* old maintainer no pystache activity in 7 yrs (but still some on github)
* way too big for a patch

Comment (sarnold) [reply recipe/meta.yaml:44]: This is very clean now, with almost complete test coverage:  https://github.com/sarnold/pystache

Comment (ocefpaf) [reply recipe/meta.yaml:44]: @sarnold the core members are not very keen on "keep-alive" forks that are not the canonical repo or "blessed" by the original maintainer. Ideally, forks would change the name or request upstream to became the canonical source.

Comment (sarnold) [reply recipe/meta.yaml:44]: I'm not sure what " request upstream " means exactly. I can make a PR and see what happens.

**Comment by ocefpaf** [comment]
@sarnold we just had a @conda-forge/core meeting where we discussed the case of "New Upstream." Here is what we believe is an OK-ish way to solve it. (Spoiler, yours is the most complicated one!)

1. Try to send the changes as a patch with a link to a PR upstream.
2. If upstream is non-responsive try to change the name to something that users can related, like `pystache-fork`, or `pystache-myusername`, etc.
3. If renaming is a no-no because it would cause a lot of churn downstream, try to get the upstream "blessing" for your fork.
4. If all fail we would need some evidences that upstream is not responsive or on longer interested in the project where a fork can claim to be the new source. One would be to take over the PyPI name, we would trust PyPI's judgment. Another one would be to open an issue upstream with say, 3, and come back after a certain amount of time they don't respond (probably with more data like, last PR merged, active contributions, etc).

Does that make sense?

PS: we will write that as docs soon, just polishing the details...

**Comment by ocefpaf** [comment]
Oh. I forgot, there is still an issue with claiming a name. We may have the license for the code but a name may be trademarked or something, so we will have to be careful if the only option is the last one.

**Comment by sarnold** [comment]
Still no upstream answer, and I have no idea how to go about getting pypi "blessing" to push new versions there.  Have you guys done that before?

Also I'm starting to rethink the renaming option.  Would that mean only the conda package name would change?

**Comment by ocefpaf** [comment]
> Also I'm starting to rethink the renaming option. Would that mean only the conda package name would change?

Yep. It would be a new package with its own PyPI and conda namespace.

**Comment by sarnold** [comment]
I think we have some movement on this one; I'm pushing my changes to another maintainer who is supposed to take over the pypi side.  Stay tuned!

**Comment by ocefpaf** [comment]
> I think we have some movement on this one; I'm pushing my changes to another maintainer who is supposed to take over the pypi side. Stay tuned!

Awesome! Please ping us if you need anything.

**Comment by sarnold** [comment]
Background: https://github.com/defunkt/pystache/issues/192

New PR: https://github.com/PennyDreadfulMTG/pystache/pull/1

**Comment by xylar** [comment]
I'm closing this because I think it has been addressed in the intervening 3 years.

**Pull request closed without merging**